compiler/z/codegen/S390Peephole.cpp

```diff
@@ -1283,7 +1283,7 @@ bool
 TR_S390PostRAPeephole::ConditionalBranchReduction(TR::InstOpCode::Mnemonic branchOPReplacement)
    {
    // This optimization relies on hardware instructions introduced in z13
-   if (!TR::Compiler->target.cpu.getSupportsArch(TR::CPU::z13))
+   if (!TR::Compiler->target.cpu.isAtLeast(OMR_PROCESSOR_S390_Z13))
```

Comment:
This is going to sound a little selfish however these changes are going to cause problems for #5235 since that PR heavily modifies this file and has many commits (50+). Making this change here would mean I would have to rebase most of those commits to absorb the changes here, which is very error prone.

Given we are not removing the `getSupportsArch` APIs just yet, how would you feel if we reverted the changes in this file only, and make them after #5235 is merged? I tried checking out your branch and rebasing my PR on top and it was not easy. I tried after reverting the changes in this file only and the rebase was clean.

Reply:
Sure we can do that. Sounds reasonable to me.

Reply:
changes for S390Peephole.cpp: https://github.com/eclipse/omr/pull/5255